Source/Scene/GlobeSurfaceTile.js

```diff
@@ -120,6 +142,28 @@ define([
         }
     });
 
+    GlobeSurfaceTile.prototype.getBvh = function(tile, terrainProvider) {
+        if (this._bvh === undefined) {
+            var terrainData = this.terrainData;
+            if (terrainData !== undefined && terrainData.bvh !== undefined) {
```

Comment:
Throughout, should these be using `!defined()` for consistency even if it is old school?

Reply:
> Throughout, should these be using !defined() for consistency even if it is old school?

I'm torn. I mean, checking for null when it will only ever be undefined is a waste of time, right? Not like this is public API stuff. But yeah, we usually use `defined()` and I didn't have a really good reason to do differently here.